src/main/java/com/solutions/shop/controller/UserController.java

```diff
@@ -3,28 +3,28 @@
 import com.solutions.shop.model.User;
 import com.solutions.shop.service.UserService;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.RestController;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.*;
 import org.springframework.http.ResponseEntity;
 import org.springframework.http.HttpStatus;
 
+import java.util.Optional;
+
 /* Logic of processing client requests. */
 
 @RestController
 public class UserController {
     @Autowired
     private UserService userService;
 
-    @PostMapping
+    @PostMapping(value = "user/{login},{password}")
     public ResponseEntity<?> create(@RequestBody User user) {
         userService.createUser(user);
         return new ResponseEntity<>(HttpStatus.CREATED);
     }
-/*
-    @PostMapping
-    public ResponseEntity<User> create(@RequestBody User user){
-        return ResponseEntity.status(HttpStatus.CREATED).body(userService.createUser(user));
+
+    @GetMapping(value = "user/{login},{password}")
+    public ResponseEntity<User> findUserByLogin(@PathVariable("login") String login,
+                                             @PathVariable("password") String password){
+        return ResponseEntity.status(HttpStatus.OK).body(userService.findUser(login, password));
```

Comment:
1) Change entity User result to UserDTO

2)         return ResponseEntity
                .status(HttpStatus.OK)
                .body(userService.findUser(login, password));
                
3) Services **cannot operate with entities.** Changes all services to DTOs!